### PR TITLE
[Android] connection processor possible try catch

### PR DESCRIPTION
**Pull request by arifBurakDemiray**

### CODE REVIEW

**Comment by arifBurakDemiray** [comment]
[[Android] connection processor possible try catch](https://trello.com/c/JQ8HHBCp/1057-android-connection-processor-possible-try-catch)

**Review by peterBrxwn — APPROVED** [review]

File: sdk/src/main/java/ly/count/android/sdk/ConnectionProcessor.java [hunk 1]
```diff
@@ -191,14 +191,20 @@ synchronized public URLConnection urlConnectionForServerRequest(String requestDa
         }
 
         //calculating header field size
-        int headerIndex = 0;
-        while (true) {
-            String key = conn.getHeaderFieldKey(headerIndex);
-            if (key == null) {
-                break;
+        try{ //just after init, because of background operations, this might fail
```
Comment (peterBrxwn) [inline sdk/src/main/java/ly/count/android/sdk/ConnectionProcessor.java:194]: comments should be in new line... change `try{` to `try {`

**Review by peterBrxwn — APPROVED** [review]

File: sdk/src/main/java/ly/count/android/sdk/ConnectionProcessor.java (hunk 1 above)
Comment (arifBurakDemiray) [reply sdk/src/main/java/ly/count/android/sdk/ConnectionProcessor.java]: done

**Review by ijunaid — APPROVED** [review]

**Review by turtledreams — CHANGES_REQUESTED** [review]
can you add a changelog pls 🙏

**Review by ArtursKadikis — APPROVED** [review]

**Pull request merged**